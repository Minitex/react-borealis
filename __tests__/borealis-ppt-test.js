import React from 'react';
import renderer from 'react-test-renderer';
import BorealisPPT from '../src/borealis-ppt';

describe('BorealisPPT Tests', () => {
  it('renders a PowerPoint', () => {
    const config = {
      ppt: {
        transcript: {
          texts: [],
          label: 'PowerPoint',
        },
        thumbnail: 'http://reflections.mndigital.org/utils/getthumbnail/collection/p16022coll17/id/827',
        src: 'http://reflections.mndigital.org/utils/getdownloaditem/collection/p16022coll17/id/827/filename/828.pptx',
      },
    };
    const component = renderer.create(<BorealisPPT config={config} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
