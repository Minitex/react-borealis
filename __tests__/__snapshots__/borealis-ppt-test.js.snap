exports[`BorealisPPT Tests renders a PowerPoint 1`] = `
<div
  className="ppt-download">
  <a
    href={undefined}>
    <img
      alt="Microsoft PowerPoint"
      src={undefined} />
  </a>
  <a
    href={undefined} />
</div>
`;
